components/pkg-export-kubernetes/src/manifest.rs

```diff
@@ -91,9 +95,26 @@ impl Manifest {
         };
         let name = format!("{}-{}", pkg_ident.name, version_suffix);
 
-        let image = match matches.value_of("IMAGE_NAME") {
+        let image_name = match matches.value_of("IMAGE_NAME") {
```

Comment:
One way to kill another `match`, which I think would work:

```
value_of("IMAGE_NAME")
.map(|i| i.to_owned())
.or_else(<None block>)
```

Not sure if it's pushing it too far though 🤷‍♂️

Reply:
I wonder what's wrong with `match`…

Or are we playing perlgolf? Err, I mean - rustgolf?

Reply:
not wrong with match really and it's one of the coolest thing after sliced bread but I've seen rust devs prefering Options & Result method chaining over match and usually it makes things very compact so why not. However, there is no reason to be go on a "must kill all match statements" spree. :)

Reply:
Agreed, I've also had the impression that chaining is more idiomatic - part of the charm of rust are these functional programming inspired bits 👅